themes/bootstrap3/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -40,7 +51,7 @@
             </a>
           <?php endif; ?>
         <?php else: ?>
-          <?php /* Begin Unavailable Items (Recalls) */ ?>
+          <?php /* Begin Unavailable Items (Holds/Recalls) */ ?>
```

Comment:
Why the changed comment here? I think in VuFind's simplistic view, available items get holds and unavailable items get recalls... and that's why "holds" wasn't part of this comment. Has something changed?

I also notice that the equivalent line in extended.phtml was NOT changed, so if we do keep this change, we should make the parallel edit there.

Reply:
That was a mistake, now reverted (though it bugs me a bit that it really is the hold function everywhere and it's up to the ILS to decide if it's handled as a recall).

Reply:
Agreed; adding more granularity to that might make sense at some point, but somehow we've gotten this far without anyone raising a use case where the distinction was crucial.